softlayer/baremetal/baremetal_creator_test.go

```diff
@@ -0,0 +1,40 @@
+package baremetal_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	boshlog "github.com/cloudfoundry/bosh-agent/logger"
+	common "github.com/maximilien/bosh-softlayer-cpi/common"
+	bm "github.com/maximilien/bosh-softlayer-cpi/softlayer/baremetal"
+	fakeslclient "github.com/maximilien/softlayer-go/client/fakes"
+)
+
+var _ = Describe("BaremetalCreator", func() {
```

Comment:
Can we have more tests for the BM creator? Seems like at least one test to verify bad parameters would be needed?